Sokoliuk/HW5/HW5_task_02_and_03.py

```diff
@@ -0,0 +1,57 @@
+print("Hello, to start a program press one of the folowing:\n \
+1 to print Fibonaci numbers;\n \
+2 to calculate the factorial;\n \
+3 to exit a program.\n")
+
```

Comment:
this code is hard to read
I advise you to split this code into three separate parts
input_number = int(input("Input natural number: "))

fact = 1

if input_number == 1 or input_number == 0:
    print("Factorial {}! =".format(input_number), fact)
elif input_number < 0:
    print("Factorial non positive number don't exist")
else:
    for item in range(1,input_number + 1):
        fact *= item
    print("Factorial {}! =".format(input_number), fact)



n = int(input('enter some number from fibonacci list= '))

fibonacci_numbers = [0, 1]

for i in range(2,n):
        fibonacci_numbers.append(fibonacci_numbers[i-1]+fibonacci_numbers[i-2])

Reply:
and you need to resolve conflict